pgvector-14.yaml

```diff
@@ -1,5 +1,5 @@
 package:
-  name: pgvector
+  name: pgvector-14
```

Comment:
Any more info here? I see in the description we talk about needing to compile against different postgres versions but i'm not sure I understand the change.

looks like we'll break any dependent images / packages that used 'pgvector' as the package name, and don't see any package withdraws or provides?